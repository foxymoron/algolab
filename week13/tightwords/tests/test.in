10
4 5
0 8
3 5
3 2
0 5
8 5
9 3
7 9
8 6
5 4
