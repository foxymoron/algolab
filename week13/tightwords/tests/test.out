259
1
178
10
1
583
80
33942
1679
122
